spec/models/database_backup_spec.rb

```diff
@@ -1,13 +1,14 @@
 describe DatabaseBackup do
   context "region" do
+    let(:region) { FactoryGirl.create(:miq_region, :region => 3) }
```

Comment:
I think this should be `let!` since the method being called depends on a region record which is now being created not intentionally, but as a side effect of the test

Reply:
👍 thanks